plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -718,8 +809,53 @@ protected String renameTableSql(String catalogName, String remoteSchemaName, Str
                 quoted(catalogName, newRemoteSchemaName, newRemoteTableName));
     }
 
+    private RemoteTableName constructHelperTable(ConnectorSession session, Connection connection, JdbcOutputTableHandle handle, Set<Long> pageSinkIds)
+            throws SQLException
+    {
+        checkArgument(handle.getPageSinkIdColumnName().isPresent(), "Output table handle's pageSinkIdColumn is empty");
```

Comment:
can be `verify` instead `checkArgument` - as this is an assertion on condition checked before really.